### ISSUE TITLE
Misskeyについて: 「主なコントリビューター」を現行のMisskey Devメンバー一覧にしたい

### ISSUE THREAD

**Issue by EbiseLutica** [syuilo/Misskey#10965]
既に長いことMisskey本家プロジェクトに関わっていない人が列挙されているのもどうかと思う。

**Comment by acid-chicken** [comment]
どうかと思う理由がわからない（重要な貢献は永く評価されるべきでは？）

**Comment by EbiseLutica** [comment]
> 重要な貢献は永く評価されるべき

これはそうだけど、じゃあ私が入っていないのもおかしなわけで…基準が曖昧なのでどちらにせよ方針決めたい

**Comment by acid-chicken** [comment]
> > 重要な貢献は永く評価されるべき
> 
> これはそうだけど、じゃあ私が入っていないのもおかしなわけで…基準が曖昧なのでどちらにせよ方針決めたい

自身の話に差し当たって他人を引き合いに出すのは少し礼に欠けるのではと思うのと、また必ずしも misskey-dev メンバーが主な貢献者の部分集合とも言えるか現状では怪しいとも思う（まあそれはそれとして別項で載せてもいいとは思うけど）

**Comment by EbiseLutica** [comment]
> 自身の話に差し当たって他人を引き合いに出すのは少し礼に欠けるのではと思う

そうですね、配慮が欠けていました。この問題の本質としては  `/about-misskey` にある「貢献者」に選ばれる基準が曖昧なことを挙げたく思っているので、それを定めたい（私たちの裁量ではなくしゅいろの裁量かもしれないけど）です

それはそれとして、contributor = member ではないのも事実なので、メンバーの枠も欲しい

**Comment by syuilo** [comment]
良い感じの基準決めるのは難しく感じた
Misskey Devメンバーを表示する場合は今はいないけど過去に大きく貢献した人が含まれないのは微妙な感じがする

**Comment by syuilo** [comment]
そういった人を含める場合でも大きな貢献の基準が決められなさそう

**Comment by syuilo** [comment]
いっそセクション自体消すとか

**Comment by syuilo** [comment]
もしくは主なコントリビューターという扱いではなくてDevチームというふうにして現行のMisskey Devメンバーだけ載せても良いかも

**Comment by u1-liquid** [comment]
モチベに繋がるんだったらメンバー枠を追加するのはわるくなさそう

**Comment by EbiseLutica** [comment]
ぶっちゃけどんくらい貢献してくれたかを私たちが測ってページに追加するより、GitHub APIにつなぐでもして自動生成するくらいじゃないと難しそうだしね。

**Comment by u1-liquid** [comment]
自動生成にするとContributor上位ランクにbotが…👀

**Comment by syuilo** [comment]
ちなみにFactorioだとメンバー一覧に過去在籍していた人も但し書き付きで載ってるのでそんな感じにするのもあり
https://www.factorio.com/game/about

**Comment by EbiseLutica** [comment]
> 自動生成にするとContributor上位ランクにbotが…👀

botとdevメンバーは削りましょう:woozy_face:

**Comment by syuilo** [comment]
> ちなみにFactorioだとメンバー一覧に過去在籍していた人も但し書き付きで載ってるのでそんな感じにするのもあり
> https://www.factorio.com/game/about

但し書きというか別セクションだった

**Comment by sasagar** [comment]
バージョン単位で区切ってコントリビューターをリストにするパターンはあったりしそう。  
（それをMisskeyでやるかどうかは別として、ここまでの問題点を解消する一つの考え方としてありなのかもなあと。）
